app/src/pages/common/uiExtensionPage/uiExtensionPage.jsx

```diff
@@ -16,14 +16,18 @@
 
 import React, { useState } from 'react';
 import PropTypes from 'prop-types';
-import { PageSection } from 'layouts/pageLayout';
+import { PageHeader, PageLayout, PageSection } from 'layouts/pageLayout';
 import { ExtensionLoader, extensionType } from 'components/extensionLoader';
 import { Header } from 'pages/inside/projectSettingsPageContainer/header';
 import classNames from 'classnames/bind';
 import styles from './uiExtensionPage.scss';
 
 const cx = classNames.bind(styles);
 
+const newExtensionsScopesMap = {
+  billing_saas_plugin: true,
+};
```

Comment:
Some notes regarding this.
1. We don't mention any particular plugins in the core UI.
2. It's better to handle this based on the extensions metadata. Just add a property `"newLayout": true` or `"layoutType": "new"` to the proper extension in the plugin's `metadata.json` file.

Later, when we'll migrate all of our plugins to the new layout type, this check will be removed.